### PR TITLE
added bottom navigation for mobile screens

### PR DESCRIPTION
**Pull request by nikhiljugale007**
Removed sidebar navigation for mobile screens and implemented bottom navigation.
Files to check:
App.js
App.css
MobileNavigation.jsx
MobileNavigation.css

### CODE REVIEW

**Review by rajaranjanmr — APPROVED** [review]
suggestion : MobileNavigation.css is empty file if not needed you can remove it.
Rest seems to be good you can merge

**Comment by nikhiljugale007** [comment]
> suggestion : MobileNavigation.css is empty file if not needed you can remove it. Rest seems to be good you can merge


Made changes in new commit.

File: src/App.css [hunk 1]
```diff
@@ -1,7 +1,11 @@
+.home-page-container {
+	display: grid;
+	grid-template-rows: auto 1fr auto;
+}
 .content-section {
 	display: grid;
 	grid-template-columns: 200px 1fr;
```
Comment (sri-rishi) [inline src/App.css]: Suggestion: can use rems here

File: src/App.css [hunk 2]
```diff
@@ -15,10 +19,20 @@
...
 	overflow: scroll;
 }
+.mobile-nav {
+	background-color: var(--hover-color);
+	position: absolute;
+	bottom: 0;
+	width: 100%;
+	display: none;
+}
 @media only screen and (max-width: 768px) {
```
Comment (sri-rishi) [inline src/App.css]: Suggestion: can use ems for media

**Review by Sahil25061999 — APPROVED** [review]
The ui looks good, navigation looks great, nice coding convention used, liked the css class naming used.

**Review by sri-rishi — APPROVED** [review]
Code looks good, one thing if there is a empty file and you are planning to working on that in future, then please add a comment about what is to come there , it will help reviewer to know that this is not unnecessary file

File: src/App.css (hunk 1 above)
Comment (nikhiljugale007) [reply src/App.css]: used rems instead of pixels in new commit.

File: src/App.css (hunk 2 above)
Comment (nikhiljugale007) [reply src/App.css]: replace with em in new commit.

**Pull request merged**